src/style.css

```diff
@@ -1,3 +1,74 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the 'Roboto' font is correctly loaded in your HTML file. You have linked to Google Fonts in your HTML, so this should work as expected. Just make sure the link is correct and the font is available.